#List of key value pairs for sts_behavior_tree color updates
Header header
bt_draw_event[] updateList
